mir_perception/mir_object_recognition/ros/scripts/pc_object_recognizer_node

```diff
@@ -158,8 +158,9 @@ if __name__ == '__main__':
     model_dir = rospy.get_param("~model_dir")
     dataset = rospy.get_param("~dataset")
 
-    object_recognizer = PointcloudObjectRecognizer(
-        model, model_id, dataset, model_dir)
-    rospy.loginfo('\033[92m'+"PCL Recognizer is ready using %s , model: %s ",
-                  model, model_id)
+    # commented as we'are not using it
```

Comment:
we should remove it completely (i.e. not commented out), until we know what we're doing for 3D recognition